### PR TITLE
Implement isna

### PR DESCRIPTION
**Pull request by 1e-to**

### CODE REVIEW

File: hpat/tests/test_series.py [hunk 1]
```diff
@@ -1265,6 +1265,14 @@ def test_impl(S, values):
         values = {'b', 'c', 'e'}
         pd.testing.assert_series_equal(hpat_func(S, values), test_impl(S, values))
 
+    def test_series_isna(self):
```
Comment (akharche) [inline hpat/tests/test_series.py]: We already have such test in test_series: test_series_isna1. Please, use it or specify name of these tests like: test_series_isnan_int and test_series_isnan_float

File: hpat/tests/test_series.py [hunk 2]
```diff
@@ -1265,6 +1265,14 @@ def test_impl(S, values):
...
+        def test_impl(S):
+            return S.isna()
+        hpat_func = hpat.jit(test_impl)
+
+        S = pd.Series([0, np.NaN, -3])
```
Comment (akharche) [inline hpat/tests/test_series.py]: Let's follow common definition: np.nan

File: hpat/datatypes/hpat_pandas_series_functions.py [hunk 3]
```diff
@@ -356,6 +357,44 @@ def hpat_pandas_series_append_impl(self, to_append):
...
+         returns :obj:`pandas.Series` object
+    """
+
+    _func_name = 'Method isna().'
+
+    if not isinstance(self, SeriesType):
+        raise TypingError(
+            '{} The object must be a pandas.series. Given self: {}'.format(_func_name, self))
+
+    if not isinstance(self.dtype, (types.Integer, types.Float)):
```
Comment (akharche) [inline hpat/datatypes/hpat_pandas_series_functions.py]: I think, we can add implementation for Strings too

**Comment by akharche** [comment]
Please, define commit message more precisely. Now it is not obvious, which type this method is implemented for 
Ex: Implement series.isna

File: hpat/datatypes/hpat_pandas_series_functions.py [hunk 4]
```diff
@@ -356,6 +357,65 @@ def hpat_pandas_series_append_impl(self, to_append):
     return hpat_pandas_series_append_impl
 
 
+@overload_method(SeriesType, 'isna')
+def hpat_pandas_series_isna(self):
+    """
+    Pandas Series method :meth:`pandas.Series.append` implementation.
```
Comment (kozlov-alexey) [inline hpat/datatypes/hpat_pandas_series_functions.py]: Wrong method name

File: hpat/datatypes/hpat_pandas_series_functions.py [hunk 5]
```diff
@@ -356,6 +357,65 @@ def hpat_pandas_series_append_impl(self, to_append):
...
+
+    Parameters
+    -----------
+    self : :obj:`pandas.Series` object
+               input argument
+
+    Returns
+    -------
+    :obj:`pandas.Series`
+         returns :obj:`pandas.Series` object
```
Comment (kozlov-alexey) [inline hpat/datatypes/hpat_pandas_series_functions.py]: Can you please describe what this output series contains, e.g. 'that contains bool values indicating if corresponding element of self is NaN"

File: hpat/datatypes/hpat_pandas_series_functions.py [hunk 6]
```diff
@@ -356,6 +357,65 @@ def hpat_pandas_series_append_impl(self, to_append):
...
+    """
+
+    _func_name = 'Method isna().'
+
+    if not isinstance(self, SeriesType):
+        raise TypingError(
+            '{} The object must be a pandas.series. Given self: {}'.format(_func_name, self))
+
+    if isinstance(self.dtype, (types.Integer, types.Float)):
```
Comment (kozlov-alexey) [inline hpat/datatypes/hpat_pandas_series_functions.py]: types.Integer - do we really handle integer series? I thought there's no such thing as NaN in integer world.
Can you please check that numpy array does have int dtype?

File: hpat/datatypes/hpat_pandas_series_functions.py (hunk 4 above)
Comment (1e-to) [reply hpat/datatypes/hpat_pandas_series_functions.py]: I close this PR, look new one

File: hpat/tests/test_series.py (hunk 2 above)
Comment (kozlov-alexey) [inline hpat/tests/test_series.py]: Let's add np.inf as well and see if it's handled correctly.

File: hpat/datatypes/hpat_pandas_series_functions.py [hunk 7]
```diff
@@ -356,6 +357,65 @@ def hpat_pandas_series_append_impl(self, to_append):
...
 
 
+@overload_method(SeriesType, 'isna')
+def hpat_pandas_series_isna(self):
+    """
+    Pandas Series method :meth:`pandas.Series.append` implementation.
+
+    .. only:: developer
+
+        Test: python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_isna
```
Comment (kozlov-alexey) [inline hpat/datatypes/hpat_pandas_series_functions.py]: Test for string type -  test_series_str_isna1- should be mentioned here too

**Pull request closed without merging**